compiler/nvrtc_builder.cc

```diff
@@ -21,9 +22,10 @@ namespace chainer_compiler {
 namespace {
 
 std::string CleanseIdent(const std::string& s, const char* prefix = "v_") {
+    std::locale loc;
     std::string o = prefix;
     for (char c : s) {
-        if (std::isalnum(c)) {
+        if (std::isalnum(c, loc)) {
```

Comment:
Not sure why we need this change: https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/isalnum-iswalnum-isalnum-l-iswalnum-l?view=vs-2019